src/ru/artyushov/jmhPlugin/configuration/JmhConfigurationProducer.java

```diff
@@ -31,4 +31,8 @@ protected abstract boolean setupConfigurationFromContext(JmhConfiguration config
 
     @Override
     public abstract boolean isConfigurationFromContext(JmhConfiguration jmhConfiguration, ConfigurationContext configurationContext);
+
+    String createProgramParameters(String generatedParams, String defaultParams) {
+        return generatedParams + " " + defaultParams;
```

Comment:
Thanks for nullcheck, I always forget (`Optional` ftw!).
What exactly is benefit of have it static?